v63003/instream.csh

```diff
@@ -76,6 +76,14 @@ if ("pro" == "$tst_image") then
 	setenv subtest_exclude_list "$subtest_exclude_list gtm8839"
 endif
 
+if ("dbg" == "$tst_image") then
```

Comment:
Add comment "Disable debug builds for tests that are timing sensitive"

Reply:
With the latest min-after-max change, I think even dbg builds will work fine with this test so please re-enable it in the test.